manifests/d/Discord/Discord/1.0.9144/Discord.Discord.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Discord.Discord
+PackageVersion: 1.0.9144
+InstallerType: exe
+Scope: user
+InstallModes:
+- silent
+- interactive
+UpgradeBehavior: deny
+Protocols:
+- discord
+RequireExplicitUpgrade: true
+ReleaseDate: 2024-04-29
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.discordapp.net/distro/app/stable/win/x64/1.0.9144/DiscordSetup.exe
+  InstallerSha256: 718DF871D991272884C58D29FAA197088A2D305ED662ADAF81492CC1728F9902
+- Architecture: x86
+  InstallerUrl: https://dl.discordapp.net/distro/app/stable/win/x86/1.0.9044/DiscordSetup.exe
+  InstallerSha256: BBCE4FF7B2BD30D103A018E81E7FBB56AA41A0EC938CDFF2E5D2EA98E96F59B6
```

Comment:
I didn't catch this at the time, but this will have to be updated. Discord seems to be giving their `x64` and `x86` architectures different versions.